docs/rofl/features/proxy.mdx

```diff
@@ -41,27 +41,52 @@ Proxy:
 The proxy behavior can be configured using annotations in
 your `compose.yaml` file.
 
-The annotation key is `net.oasis.proxy.ports.<published_port>.mode`.
+### Overview
 
-Supported modes are:
+Each annotation follows this general format:
 
-- `terminate-tls` (default): The proxy terminates the TLS connection and
-  forwards the unencrypted traffic to your container.
-  This is suitable for HTTPS services.
-- `passthrough`: The proxy forwards the raw TCP connection to your container.
-  This is suitable for services that handle their own TLS or
-  use other TCP-based protocols.
-- `ignore`: The proxy will ignore this port, and it will
-  not be exposed publicly.
+`net.oasis.proxy.ports.<published_port>.<setting>: <value>`
 
-Example of configuring a port for TCP passthrough:
+Where:
 
-```yaml title="compose.yaml"
+`<published_port>` — the external port exposed in your compose.yaml
+
+`<setting>` — the specific proxy configuration (e.g., mode, custom_domain)
+
+### Example Configuration
+
+The following example configures a service to use TCP passthrough with a custom domain:
+
+```yaml
+title="compose.yaml"
 services:
   myservice:
     image: docker.io/my/service:latest
     ports:
       - "8080:8080"
+      - "80:80"
     annotations:
       net.oasis.proxy.ports.8080.mode: passthrough
+      net.oasis.proxy.ports.80.custom_domain: mydomain.com
 ```
+
+Explanation:
+- The container exposes ports 8080 and 80.
+- The proxy forwards the raw TCP connection to your container (mode: passthrough).
+- Requests to mydomain.com are routed to this service (custom_domain: mydomain.com).
```

Comment:
This is maybe a bit to simlpified. See my other comment about `custom_domain` setting.